app/ts/background/backgroundUtils.ts

```diff
@@ -99,6 +99,21 @@ export async function setExtensionIcon(details: browser.action._SetIconDetails)
 	}
 }
 
+export async function setExtensionTitle(details: browser.action._SetTitleDetails) {
+	try {
+		const manifest = browser.runtime.getManifest()
+		if (manifest.manifest_version === 2) {
+			await browser.browserAction.setTitle(details)
+		} else {
+			await browser.action.setTitle(details)
+		}
+		checkAndThrowRuntimeLastError()
+	} catch {
+		console.warn('failed to set extension title')
+		console.warn(details)
+	}
```

Comment:
Can you move this try/catch block to `setInterceptorIcon` or whatever function calls that?  Generally speaking, I find it preferable to put generalized error handling as high up the stack as possible, usually where there is an async function given as a sync callback.